libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -361,7 +361,8 @@ private MapRouteLine buildMapRouteLine(@NonNull MapView mapView, @NonNull Mapbox
         styleRes,
         belowLayer,
         layerProvider,
-        new MapRouteSourceProvider()
+        new MapRouteSourceProvider(),
+        null
```

Comment:
We should be able to pass the callback via the `NavigationMapRoute`'s constructor as well.